src/Microsoft.Azure.SignalR.Common/Auth/JwtData.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Security.Claims;
+using System.Globalization;
+using Microsoft.IdentityModel.Logging;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.SignalR
+{
+    /// <summary>
+    /// A unified and simplified version of class JwtHeader and class JwtPayload
+    /// </summary>
+    /// Ref:
+    ///     JwtHeader.cs: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/799a162276cb699c6634f12077ab7af0646cf710/src/System.IdentityModel.Tokens.Jwt/JwtHeader.cs
```

Comment:
can we still use the same file structure as 2 files but simplify?

Reply:
fixed